install.py

```diff
@@ -97,7 +97,11 @@ def check_distribution_compatibility():
 def get_distribution_info():
 	# return distribution name and major version
 	if platform.system() == "Linux":
-		current_dist = platform.dist()
+        if (platform.python_version_tuple()[0] + platform.python_version_tuple()[1]) == '2.7':
```

Comment:
If the it is python version 2.7 use  platform.dist() for python3 use dist.linux_distribution()